zenoh-protocol/src/link/locator.rs

```diff
@@ -63,19 +71,35 @@ pub enum Locator {
     Tcp(SocketAddr),
     #[cfg(feature = "udp")]
     Udp(SocketAddr),
+    #[cfg(all(feature = "unix", target_family = "unix"))]
+    Unix(PathBuf),
```

Comment:
I suggest to call the locator UnixDomainSocket and not simply Unix, alternatively we can use Uds.